common/internal_model/src/test/scala/uk/ac/wellcome/models/work/internal/AvailabilityTest.scala

```diff
@@ -0,0 +1,47 @@
+package uk.ac.wellcome.models.work.internal
+
+import org.scalatest.funspec.AnyFunSpec
+import org.scalatest.matchers.should.Matchers
+import uk.ac.wellcome.models.work.generators.{ItemsGenerators, WorkGenerators}
+
+class AvailabilityTest
+    extends AnyFunSpec
+    with Matchers
+    with WorkGenerators
+    with ItemsGenerators {
+  describe("Availabilities.forWorkData") {
+    it(
+      "adds Availability.Online if there is a digital location with an Open or OpenWithAdvisory access status") {
+      val openWork = denormalisedWork().items(
+        List(createDigitalItemWith(accessStatus = AccessStatus.Open)))
+      val openWithAdvisoryWork = denormalisedWork().items(
+        List(
+          createDigitalItemWith(accessStatus = AccessStatus.OpenWithAdvisory)))
```

Comment:
I don't think it's a massive issue, but note that digital locations with LicensedResources also get Availability.Online, as of https://github.com/wellcomecollection/catalogue/pull/1431